requirements.txt

```diff
@@ -0,0 +1,17 @@
+jcconv
```

Comment:
Shouldn't they reference frozen versions ones (e.g.: Flask-Babel==0.12.2) to manage in a better way releases ?

Reply:
Hi, thanks for the review, you're right in an ideal world all the dependencies must be precisely frozen, which might require to create one branch per version or maybe other things, but with the time we have internally to manage the project it seems difficult to maintain, I rather suggest that we keep our efforts to maintain the project as it exists in this Github repository up to date with the latest dependencies. As such I added in a last commit script "build.sh" which should allow everyone to test a build easily a debian package just from this repository and without special knowledge in the debian packaging. I know this is not the state of the art of debian packaging, but it seemed to me best to be able to offer a low cost debian package for this project.
Thank you very much for your interest in the project!

Reply:
> EDIT: For a complete answer the dependencies are frozen at the time of debian pakcage build because all the dependencies are included (wheel) in the debian package and installed in an isolated virtualenv, so normally it is reproducible contrary to the old packaging system (post-install)

That's true for that case. But some maybe want to build from source (e.g.: We did it for windows environment with nsis packaging or for development).